circle.yml

```diff
@@ -1,7 +1,8 @@
 dependencies:
-    override:
-        - pip install -r dev_requirements.txt
+  override:
+    - pip install -r dev_requirements.txt
 
 test:
-    override:
-        - python test_project/manage.py test
+  override:
+    - sudo pip install .
```

Comment:
The old circle config didn't actually run any tests. it does now, but the `queue_fetcher` module isn't visible to the test, so this installs the package first so it can be imported. Probably doesn't need `sudo` though, I'll fix that